Source/Lib/DPX/DPX.cpp

```diff
@@ -197,6 +228,32 @@ bool dpx::Parse()
     if (Get_X4() != 0)
         return Error("End-of-line padding");
     //uint32_t EndOfImagePadding = Get_X4(); //We do not rely on EndOfImagePadding and compute the end of content based on other fields
+    
+    if (FrameRate)
+    {
+        Buffer_Offset = 1724;
+        double FrameRate_Film = Get_XF4(); // Frame rate of original (frames/s) 
+        Buffer_Offset = 1940;
+        double FrameRate_Television = Get_XF4(); // Temporal sampling rate or frame rate (Hz)
+
+        // Integrity of frame rate
+        if (FrameRate_Film && FrameRate_Television && FrameRate_Film != FrameRate_Television)
+            return Error("\"Frame rate of original (frames/s)\" not same as \"Temporal sampling rate or frame rate (Hz)\"");

```

Comment:
Why do you make a difference between `frame/s` and `Hz`?  I would use the same terminology, because Hz means exactly s<sup>-1</sup>.

Reply:
I don't, this is the exact terminology in DPX specs, and I just copy specs lines for informing which line of the spec I read.

Reply:
Your comment should be addressed to SMPTE ;-)